Assets/MRTK/Core/Utilities/StandardShader/ClippingPrimitive.cs

```diff
@@ -314,9 +314,12 @@ protected virtual void UpdateRenderers()
             for (int i = renderers.Count - 1; i >= 0; --i)
             {
                 var _renderer = renderers[i];
-                if (Application.isPlaying && _renderer == null)
+                if (_renderer == null)
```

Comment:
Looks sane to me, if you haven't already could you sanity check adding and removing renderers while not playing via the inspector? (I've not pulled this down to try myself.)

Reply:
Adding works, but I am not sure what you mean by removing renderers as I believe the list gets automatically populated when its size is less than the number of renderers in the container. If you mean deleting GameObjects with renderers in the container then things have been verified to work in that case as well.

Reply:
Excellent, that's what I was asking about. Thanks!